rosco-web/config/rosco.yml

```diff
@@ -191,6 +191,34 @@ google:
         virtualizationSettings:
           sourceImage: ubuntu-1404-trusty-v20150909a
 
+openstack:
+  enabled: ${OS_ENABLED:false}
+  bakeryDefaults:
+    identityEndpoint: ${OS_AUTH_URL:https://openstack.yourcompany.com:5000/v3}
+    domainName: Default
+    floatingIpPool: ext_vlan1472_net
+    securityGroups: default
+    tenantName: PRD-spinnakeropenstack
```

Comment:
If you're referring to username/password, I'm not sure how we want to manage it. This way allows it to work in a reasonably secure way ... but there might be a better way.